src/WebJobs.Extensions.DurableTask/IDurableHttpMessageHandlerFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Net.Http;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// Interface used for testing Durable HTTP.
+    /// </summary>
+    public interface IDurableHttpMessageHandlerFactory
```

Comment:
Let's check to see if this interface needs to be public.

Reply:
This interface needs to be public because otherwise there's inconsistent accessibility with DurableTaskExtension. DurableTaskExtension is public and has an IDurableHttpMessageHandlerFactory argument in the constructor. Also, the other interfaces included in the DurableTaskExtension constructor are public.